lib/logstash/outputs/elasticsearch/elasticsearch-template-es5x.json

```diff
@@ -9,7 +9,7 @@
       "_all" : {"enabled" : true, "norms" : false},
       "dynamic_templates" : [ {
         "message_field" : {
-          "match" : "message",
+          "path_match" : "message",
           "match_mapping_type" : "string",
           "mapping" : {
             "type" : "string", "index" : "analyzed", "norms" : false,
```

Comment:
@talevy I missed this when I was doing #402 -- 

`"type" : "string", "index" : "analyzed", "norms" : false,`

should be

`"type" : "text", "index" : "analyzed", "norms" : false,`

`text` not `string` for 5.0